pylint/utils.py

```diff
@@ -280,12 +281,14 @@ def _register_by_id_managed_msg(self, msgid, line, is_disabled=True):
         except UnknownMessageError:
             pass
 
+    @abc.abstractmethod
```

Comment:
You probably need to make this abstract or do you rely on the fact that the classes that will use this mixin are going to be abstract themselves? Otherwise `abc.abstractmethod` won't do any validation per se.

Reply:
I originally added it because I wanted both the Pylinter class and the PluginRegistry to override the method. But you're right, we don't really need it so I've removed it.